src/js/common/components/NavButtons.jsx

```diff
@@ -81,7 +81,7 @@ export default class NavButtons extends React.Component {
     );
 
     let buttons;
-    if (path.endsWith('review-and-submit')) {
+    if (path.match(/review-and-submit(\/)?$/)) {
```

Comment:
Fun fact, `.match()` is much slower than `.endsWith()`:
![image](https://cloud.githubusercontent.com/assets/12970166/25828792/100dec0e-3407-11e7-80da-6f4e8d263789.png)
It's still so fast it doesn't really matter, but I was curious, so I wrote a little jsperf to see what the comparison is.